src/ripple/app/misc/LoadFeeTrack.h

```diff
@@ -91,49 +85,67 @@ class LoadFeeTrack
 
     std::uint32_t getLoadFactor () const
     {
-        ScopedLockType sl (mLock);
-        return std::max({ mClusterTxnLoadFee, mLocalTxnLoadFee, mRemoteTxnLoadFee });
+        std::lock_guard <std::mutex> sl (lock_);
+        return std::max({ clusterTxnLoadFee_, localTxnLoadFee_, remoteTxnLoadFee_ });
```

Comment:
That's the new `std::max` that takes an initializer list of `N` items and returns the largest item in the list.